ground/src/main/java/com/google/android/ground/ui/MarkerIconFactory.kt

```diff
@@ -69,4 +67,38 @@ class MarkerIconFactory @Inject constructor(@ApplicationContext private val cont
     // TODO: Cache rendered bitmaps.
     return BitmapDescriptorFactory.fromBitmap(bitmap)
   }
+
+  fun getClusterIcon(
```

Comment:
This method has a lot of common code with `getMarkerBitmap()` method. Can we please refactor it?

Reply:
Refactored out the shared bitmap generation bits.